packages/astro/src/runtime/client/dev-overlay/plugins/audit/a11y.ts

```diff
@@ -489,3 +563,81 @@ const a11y_non_interactive_element_to_interactive_role_exceptions = {
 	td: ['gridcell'],
 	fieldset: ['radiogroup', 'presentation'],
 };
+
+const combobox_if_list = ['email', 'search', 'tel', 'text', 'url'];
```

Comment:
Can we change snake case names to camel case?

Reply:
I can change them to camelCase. It was snake_case as it's borrowed from Svelte and that's the convention they use.

Reply:
Seems like there's a lot that uses snake_case. I think maybe we should do one PR later to convert them all at once.